app/controllers/unattended_controller.rb

```diff
@@ -85,6 +84,17 @@ def render_custom_error(status, error_message, params)
     render(:plain => "# #{message}", :status => status, :content_type => 'text/plain')
   end
 
+  def render_intermediate_template
+    ipxe_template_kind = TemplateKind.find_by(name: 'iPXE')
+    template = ProvisioningTemplate.find_by(name: 'iPXE intermediate script', template_kind: ipxe_template_kind)
```

Comment:
I guess you want to use `Setting[:intermediate_ipxe_script]` here.

Reply:
ah good catch, thanks